src/Components/Custom-Button/story.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import CustomButton from "./index";
+
+export const defaultButton = {
```

Comment:
See Drumpad's storybook for an example of how to pass props to the component. We'll be using knobs instead. 

- [ ] Use knobs instead.